scripts/Dockerfile

```diff
@@ -1,18 +1,23 @@
-FROM alpine
+FROM alpine AS build
 MAINTAINER chriseth <chris@ethereum.org>
 #Official solidity docker image
 
 #Establish working directory as solidity
 WORKDIR /solidity
+
+# Build dependencies
+ADD /scripts/install_deps.sh /solidity/scripts/install_deps.sh
```

Comment:
Why not copy the repository first?

Reply:
avoid cache invalidation each time you touch some code and wanna continuously create a new image. No need to fetch dependencies each and every time.